config/crd/bases/ran.openshift.io_clustergroupupgrades.yaml

```diff
@@ -92,7 +92,13 @@ spec:
                   timeout:
                     default: 240
                     type: integer
+                required:
+                - maxConcurrency
```

Comment:
Why is this required?

IMHO, it should default to 1.

Reply:
It was agreed on the latest discussion that we shouldn't assume what the user wants, especially since they can start an upgrade already enabled.